### PR TITLE
Feature/e UI 3576 next steps active tasks

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3576?filter=-1

### Change description ###
Add `markdown` component to render links

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -4,9 +4,9 @@ sonar.projectVersion=0.0.1
...
 sonar.sources= src/app/, src/cases/
 sonar.test.inclusions=**/*.spec.ts
-sonar.exclusions=**/node_modules/**, **/*.html, **/*.scss, **/api/test/pact/**, src/app/shared/services/mc-launch-darkly-service.ts, src/app/shared/services/rootInjectorGuard.ts
+sonar.exclusions=**/node_modules/**, **/*.html, **/*.scss, **/api/test/pact/**, **/*.module.ts
 
 client.sonar.ts.tslint.configPath=tslint.json
 client.sonar.ts.coverage.lcovReportPath=reports/tests/coverage/ng/lcov.info
-sonar.javascript.lcov.reportPaths=reports/tests/coverage/ng/lcov.info
-sonar.host.url=
+sonar.typescript.lcov.reportPaths=reports/tests/coverage/ng/lcov.info
```
Comment (rayliangatos) [inline sonar-project.properties:11]: any reason to change to typescript?

Comment (udaydenduluri33) [inline sonar-project.properties:11]: Leave it as javascript please

File: src/cases/components/case-task/case-task.component.ts [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
+  public set task(value: Task) {
+    value.description = CaseTaskComponent.replaceCaseRefVarWithCaseId(value.description, value);
+    this.pTask = value;
+  }
+
+  private static CASE_REFERENCE_VARIABLE = '${[CASE_REFERENCE]}';
+  public manageOptions: { text: string, path: string }[];
+  private pTask: Task;
+
+  public static replaceCaseRefVarWithCaseId(str: string, task: Task): string {
```
Comment (udaydenduluri33) [inline src/cases/components/case-task/case-task.component.ts:31]: Unit tests for this function please?

File: sonar-project.properties (hunk 1 above)
Comment (kuda1992) [reply sonar-project.properties:11]: I didn't make these changes I picked them out from `feature/release2-wa2-a`. Seems like you made the changes.

File: src/cases/components/case-task/case-task.component.ts (hunk 2 above)
Comment (kuda1992) [reply src/cases/components/case-task/case-task.component.ts:31]: I have a unit test for the function check the `case-task.component.spec.ts`

**Review by udaydenduluri33 — APPROVED** [review]
Approved Thanks for the Description

File: src/cases/components/case-task/case-task.component.ts [hunk 3]
```diff
@@ -1,18 +1,39 @@
...
+
+  constructor(private readonly sessionStorageService: SessionStorageService) {
+  }
+
+  public get task(): Task {
+    return this.pTask;
+  }
+
+  @Input()
+  public set task(value: Task) {
```
Comment (udaydenduluri33) [inline src/cases/components/case-task/case-task.component.ts:22]: Liked the idea of converting the task setter and changing the description

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**